recipes/ckanweb-deploy-exts.rb

```diff
@@ -250,6 +250,12 @@
 			only_if { "#{pluginname}".eql? 'ytp-comments' }
 		end
 
+		execute "Harvest CKAN ext database init" do
+			user "#{account_name}"
+			command "#{virtualenv_dir}/bin/paster --plugin=ckanext-harvest harvester initdb -c #{config_dir}/production.ini || echo 'Ignoring expected error, see https://github.com/frictionlessdata/ckanext-validation/issues/44'"
```

Comment:
Comment refers to the wrong extension. Also, does this one fail gracefully in the same way?

Reply:
sorry, copied from the example above - in which case I've removed those as well as they would be incorrect